ui_files/ui_depth_profile_params.ui

```diff
@@ -437,6 +437,9 @@
      <property name="text">
       <string>Show used efficiency files</string>
      </property>
+     <property name="checked">
+      <bool>true</bool>
+     </property>

```

Comment:
I'm not sure if this is needed because the Python code sets this anyway. This is probably fine either way.

Reply:
It comes automatically when editing the GUI.